src/main/java/org/peergos/client/NabuClient.java

```diff
@@ -77,20 +77,30 @@ public Map id() throws IOException {
         return retrieveMap("id");
     }
 
-    public List<Multihash> localRefs() throws IOException {
+    public List<Multihash> getRefs() throws IOException {
```

Comment:
Maybe a more obvious method name would be listBlockstore?

Reply:
done